### PR TITLE
BD : Add Channel B

### PR DESCRIPTION
**Pull request by BellezaEmporium**
Also titled "Voice of Bangla"

### CODE REVIEW

**Review by jkemp814 — DISMISSED** [review]
Strong 1080p stream. Audio and video smooth and clear.

File: channels/bd.m3u [hunk 1]
```diff
@@ -1,10 +1,30 @@
...
+#EXTINF:-1 tvg-id="ATNBanglaUK.bd" tvg-country="UK" tvg-language="Bengali" tvg-logo="http://atnbanglauk.tv/wp-content/uploads/2015/04/sized1.png" group-title="General",ATN Bangla UK (576p) [Geo-blocked]
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/atnbanglauk-off.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="ATNBangla.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/2/2e/ATN_Bangla.svg/207px-ATN_Bangla.svg.png" group-title="General",ATN Bangla (1080p) [Geo-blocked]
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/atnbd-8-org.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="ATNNews.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://i.imgur.com/2F1BaT2.gif" group-title="News",ATN News (720p)
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/atnws-sg.stream/playlist.m3u8
```
Comment (jkemp814) [inline channels/bd.m3u]: Works

File: channels/bd.m3u [hunk 2]
```diff
@@ -1,10 +1,30 @@
...
+#EXTINF:-1 tvg-id="BanglaVision.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://i.imgur.com/doJuPjL.png" group-title="General",Bangla Vision (1080p) [Geo-blocked]
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/banglav000.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="BijoyTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/6/63/BIJOY_TV_Logo_BD.png" group-title="General",Bijoy TV (720p)
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/bijoy00.stream/playlist.m3u8
```
Comment (jkemp814) [inline channels/bd.m3u]: Works

File: channels/bd.m3u [hunk 3]
```diff
@@ -1,10 +1,30 @@
...
+#EXTINF:-1 tvg-id="BoishakhiTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="http://www.boishakhionline.com/frontend_source/assets/images/logo.png" group-title="General",Boishakhi TV (1080p)
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/boishakhitv-org.stream/playlist.m3u8
```
Comment (jkemp814) [inline channels/bd.m3u]: Works

File: channels/bd.m3u [hunk 4]
```diff
@@ -1,10 +1,30 @@
...
+#EXTINF:-1 tvg-id="BTVChittagong.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://i.imgur.com/CMUHtv3.png" group-title="Local",BTV Chittagong (720p) [Not 24/7]
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/btvnational-ctg.stream/playlist.m3u8
```
Comment (jkemp814) [inline channels/bd.m3u]: Color test pattern

File: channels/bd.m3u [hunk 5]
```diff
@@ -1,10 +1,30 @@
...
+#EXTINF:-1 tvg-id="BTVWorld.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://i.imgur.com/1Chamk9.png" group-title="General",BTV World (720p)
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/btvbd-office-sg.stream/playlist.m3u8
```
Comment (jkemp814) [inline channels/bd.m3u]: Works

File: channels/bd.m3u [hunk 6]
```diff
@@ -1,10 +1,30 @@
...
+#EXTINF:-1 tvg-id="Channel9.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/en/thumb/f/ff/Channel9_bd.svg/350px-Channel9_bd.svg.png" group-title="Local",Channel 9 (720p) [Geo-blocked]
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/channel9hd.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="Channel24.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/9/91/Channel24logo.svg/400px-Channel24logo.svg.png" group-title="News",Channel 24 (1080p)
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/channel24-sg-e8e.stream/playlist.m3u8
```
Comment (jkemp814) [inline channels/bd.m3u]: Works

**Comment by freearhey** [comment]
There is a feeling that the channel is only broadcast on [ssh101.com](https://ssh101.com/detail.php?id=channelb) and not from Bangladesh but from US.

Maybe it would be better to create a `us_ssh101.m3u` and put this and other channels broadcast through this site there. What do you think?

**Comment by jkemp814** [comment]
Yes I agree.

**Comment by jkemp814** [comment]
Yes, I agree.

**Comment by LaneSh4d0w** [comment]
Sounds good to me. I can move all the channels I've added from this website there.

**Pull request closed without merging**